### PR TITLE
[21] Response Serialization

### PR DESCRIPTION
**Pull request by pdany1116**
#### Description

Respond with useful todo data.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ gem 'bootsnap', require: false
 # gem "image_processing", "~> 1.2"
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
```
Comment (cazacutudor) [inline Gemfile:38]: There's a lot of noise with all those comments in Gemfile. Let's clean up in another PR.

File: app/serializers/todo_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class TodoSerializer < ActiveModel::Serializer
```
Comment (cazacutudor) [inline app/serializers/todo_serializer.rb:3]: We'll need some specs to ensure the serializer it's working.

**Review by cazacutudor — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (pdany1116) [reply Gemfile:38]: https://trello.com/c/iZpVozQ9/22-clean-up-comments

File: app/serializers/todo_serializer.rb (hunk 2 above)
Comment (pdany1116) [reply app/serializers/todo_serializer.rb:3]: 37256f3ab5b3ecc5be89a7ad71f412d52576a51d

File: spec/serializers/todo_serializer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+RSpec.describe TodoSerializer do
+  subject { described_class.new(build(:todo)) }
+
+  it 'serializes the correct attributes' do
+    expect(subject.attributes.keys).to contain_exactly(:id, :title, :url, :order, :completed)
```
Comment (cazacutudor) [inline spec/serializers/todo_serializer_spec.rb:7]: The current spec it's not testing the serialization functionality, just asserts the defined attributes. That means for now, we do not know if our responses get serialized. But no worries, will get there, once we get a better understanding of what's a serializer and how will gonna help up. We'll continue to enrich the specs while working on the todo URL (https://github.com/pdany1116/todo_api/pull/23/files#diff-c13276c572c49aa1fe42895b526390c97e86d61a6663515d11a2cba899b2992dR20)

**Review by cazacutudor — APPROVED** [review]

**Pull request merged**